www/pages/register.js

```diff
@@ -18,13 +21,56 @@ class Register extends Component {
     this.state = initialState;
 
     this.handleChange = this.handleChange.bind( this );
+    this.handleChangeUsername = this.handleChangeUsername.bind( this );
+    this.handleChangeEmail = this.handleChangeEmail.bind( this );
     this.handleSubmit = this.handleSubmit.bind( this );
   }
 
   handleChange( event ) {
     this.setState({
       [ event.target.name ]: event.target.value
     });
+
+  }
+
+  handleChangeUsername( event ) {
+    this.setState({
+      [ event.target.name ]: event.target.value
+    });
+
+    checkExistingUsername( event.target.value )
```

Comment:
What do you think about calling it `userNameUniqueness`.

Reply:
I dont like it